assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -560,8 +560,8 @@ che.singleport.wildcard_domain.ipless=false
 
 # Docker image of Che plugin broker app that resolves workspace tooling configuration and copies
 # plugins dependencies to a workspace
-che.workspace.plugin_broker.init.image=eclipse/che-init-plugin-broker:v0.18
-che.workspace.plugin_broker.unified.image=eclipse/che-unified-plugin-broker:v0.18
+che.workspace.plugin_broker.init.image=eclipse/che-init-plugin-broker:v0.19.0
```

Comment:
Is there a reason we want to to use `v0.19.0`? If we use `v0.19` we would pick up bugfix releases without further modification.

Reply:
It was my fault that I specified `v0.19.0` as dockerimage and latest dockerimage tag during releasing of `che-plugin-broker` and `v0.19` is not available =(

Reply:
No worries :+1: